core/trino-parser/src/main/java/io/trino/sql/tree/DropColumn.java

```diff
@@ -27,24 +27,26 @@
 {
     private final QualifiedName table;
     private final Identifier column;
+    private final List<Identifier> field;
     private final boolean tableExists;
     private final boolean columnExists;
 
     public DropColumn(QualifiedName table, Identifier column, boolean tableExists, boolean columnExists)
     {
-        this(Optional.empty(), table, column, tableExists, columnExists);
+        this(Optional.empty(), table, column, ImmutableList.of(), tableExists, columnExists);
     }
 
-    public DropColumn(NodeLocation location, QualifiedName table, Identifier column, boolean tableExists, boolean columnExists)
+    public DropColumn(NodeLocation location, QualifiedName table, Identifier column, List<Identifier> field, boolean tableExists, boolean columnExists)
```

Comment:
`Identifier column, List<Identifier> field`

let's keep it together as a `QualifiedName`